rating/peersRatingHandler_test.go

```diff
@@ -122,6 +122,13 @@ func TestNewPeersRatingHandler(t *testing.T) {
 func TestPeersRatingHandler_AddPeers(t *testing.T) {
 	t.Parallel()
 
+	t.Run("empty peers should return work", func(t *testing.T) {
+		t.Parallel()
+
+		prh := newPeersRatingHandlerWithoutGoRoutines(createMockArgs())
+		assert.False(t, check.IfNil(prh))
+		prh.AddPeers(nil)
```

Comment:
No assert after this call?

Reply:
added check for panic. If the slice of peers is empty, there is an early return on the method